hms-app/src/components/card-details/IdeaDetails.tsx

```diff
@@ -278,6 +278,7 @@ export default function IdeaDetails(props: IProps) {
               {ideaDescription()}
             </Card.Section>
           </Spoiler>
+          {ideaCreationDate()}
```

Comment:
inside the spoiler was what i meant :)

Reply:
so directly below the description (inside spoiler and section)

Reply:
if the description is long, then the date wont be shown unless we click "show more" (?)

Reply:
![Bildschirmfoto 2023-06-07 um 08 47 38](https://github.com/knoffhoff/hms/assets/25485002/9212e28c-1733-44ea-897f-b8f8eaa04e19)
it will be like that

Reply:
yeah thats what I mean 
as not everybody may be interested in the creation date is it is no crucial information we can but in there so that people can find it when they are interested in the whole idea but will not directly see it on the card 
(maybe we could also put it somewhere into the "show details" accordion